internal/storage/config.go

```diff
@@ -44,3 +47,8 @@ type LocationConfig struct {
 	Token     string
 	Bucket    string
 }
+
+type ReportsConfig struct {
+	Bucket      *bucket.Config
+	AIPDeletion reports.AIPDeletionConfig
+}
```

Comment:
Using the internal bucket and embedding the template, we'll only need the boolean flag. But I already added a new config setting related to this in https://github.com/artefactual-sdps/enduro/pull/1402.

Considering both new settings, I'd wrap them in an "AIPDeletion" struct/section including "ApproveAMSSRequests" and "GenerateReport" boolean flags.

Reply:
Okay, sure.